### ISSUE TITLE
Missing functions

### ISSUE THREAD

**Issue by MatthieuDartiailh** [tivek/gpib_ctypes#3]
Currently gpib_ctypes follows closely what is provided by linux-gpib. However some functions required to implement low level operations in pyvisa-py are missing. Currently the missing function that I believe are required are the following:
- ibcac and ibgts: required for control atn
- ibln: or a different function to specifically address a device
- ibpct: required to implement pass control

I am not sure what is the best way to address this. I feel like we could add those and then it is up to users to know that there are departing from linux-gpib provided wrapper. I can probably find the time to put together a PR, if there is an interest.

**Comment by MatthieuDartiailh** [comment]
ping @tivek

**Comment by tivek** [comment]
@MatthieuDartiailh, linux-gpib was in recent past open to adding Python bindings for existing ib* functions. Let us add ibcac, ibgts, ibln, ibpct to gpib_ctypes. In parallel I will prepare a PR for linux-gpib's Python library so that both bindings are on par.

**Comment by tivek** [comment]
ibln binding is already there, see gpib.listener().

**Comment by MatthieuDartiailh** [comment]
Sounds good and thanks for the pointer to `listener`, I missed it.

**Comment by tivek** [comment]
How should we name the new bindings, any preferences?
Eg. ibcac -> ibcac, ibgts -> ibgts, ibpct -> ibpct (easier for those familiar with ib.h)
Or do we follow gpib.py and name the functions descriptively, eg.: ibcac -> assert_atn, ibgts -> release_atn, ibpct -> pass_control (maybe easier for newcomers)?
I prefer the latter a bit more.

**Comment by MatthieuDartiailh** [comment]
I agree the later is nicer. I went for simplicity mostly because it was only meant to be a short term solution. Maybe worth mentionning in the docstring the actual function being wrapped to ease the comparison with the GPIB specification though.
